consensus/state.go

```diff
@@ -762,18 +791,22 @@ func (cs *ConsensusState) EnterPrecommit(height int, round int) {
 
 	hash, partsHeader, ok := cs.Votes.Prevotes(round).TwoThirdsMajority()
 
-	// If we don't have two thirds of prevotes, just precommit locked block or nil
+	// If we don't have two thirds of prevotes, just precommit nil
+	// NOTE: alternatively, if we have seen a POL since our last precommit,
+	// we could precommit that
```

Comment:
nope thats me before our chat the other night. but it should say "if we have seen a POL since our last precommit, we can unlock", right?